modin/pandas/test/dataframe/test_udf.py

```diff
@@ -15,6 +15,7 @@
 import numpy as np
 import pandas
 import matplotlib
+from modin.config.envvars import MinElementsInPartition
```

Comment:
```suggestion
from modin.config import MinElementsInPartition
```
All objects from `envvars` are available from `modin.config`